cumulusci/core/keychain/encrypted_file_project_keychain.py

```diff
@@ -1,8 +1,10 @@
 import os
```

Comment:
Regarding renaming this module, have you checked to make sure no one is doing `from cumulusci.core.keychain.EncryptedFileProjectKeychain import EncryptedFileProjectKeychain`?

Reply:
I searched github and only see uses of cumulusci.core.keychain.EncryptedFileProjectKeychain, so I think this is probably okay.